src/lkl/posix-host.c

```diff
@@ -681,6 +681,13 @@ static void *host_malloc(size_t size)
 	{
 		SGXLKL_ASSERT(kernel_mem == NULL);
 		kernel_mem = enclave_mmap(0, size, 0, PROT_READ | PROT_WRITE, 0);
+        if (((intptr_t)kernel_mem) < 0)
```

Comment:
the formatting is off here due to #665. I am planning on addressing after this PR. I didn't want to muddy up the diff with a ton of formatting.